code/modules/power/tesla/coil.dm

```diff
@@ -89,13 +104,22 @@
 	anchored = 0
 	density = 1
 
+	can_buckle = TRUE
+	buckle_lying = FALSE
+	buckle_requires_restraints = TRUE
+
 /obj/machinery/power/grounding_rod/New()
 	..()
 	component_parts = list()
 	component_parts += new /obj/item/weapon/circuitboard/grounding_rod(null)
 	component_parts += new /obj/item/weapon/stock_parts/capacitor(null)
 	RefreshParts()
 
+/obj/machinery/power/grounding_rod/attack_hand(mob/user)
+	if(user.a_intent == INTENT_GRAB && user_buckle_mob(user.pulling, user, check_loc = FALSE))
```

Comment:
Um, should this be user.grabbed instead of user.pulling?
Seems odd that you have to pull someone to buckle them.

Reply:
yes.

Reply:
Change this please